scripts/not-needed.js

```diff
@@ -5,17 +5,16 @@ const path = require("path");
 
 const typingsPackageName = process.argv[2];
 const asOfVersion = process.argv[3];
-const sourceRepoURL = process.argv[4];
 const libraryName = process.argv[5] || typingsPackageName;
 
 if (process.argv.length !== 5 && process.argv.length !== 6) {
-	console.log("Usage: npm run not-needed -- typingsPackageName asOfVersion sourceRepoURL [libraryName]");
+	console.log("Usage: npm run not-needed -- typingsPackageName asOfVersion [libraryName]");
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: scripts/not-needed.js;
  Line: 11;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.